build.sbt

```diff
@@ -503,37 +500,6 @@ lazy val compilerBridge = (projectMatrix in internalPath / "compiler-bridge")
       case (2, y) if y == 11 || y == 12 => new File(scalaSource.value.getPath + "_2.11-12")
       case (2, y) if y >= 13            => new File(scalaSource.value.getPath + "_2.13")
     }.toList),
-    cleanSbtBridge := {
```

Comment:
Currently I'm trying to debug compiler bridge by making changes, and running sbt tasks. Something is caching the compiler it seems, and updates are not sticking.

Reply:
It's not going to be related to this deletion, as the values were all wrong here (e.g. sbtScalaVersion).

Reply:
https://github.com/sbt/sbt/issues/5735